js/render/draw_background.js

```diff
@@ -3,7 +3,8 @@
 var TilePyramid = require('../source/tile_pyramid');
 var pyramid = new TilePyramid({ tileSize: 512 });
 var util = require('../util/util');
-var EXTENT = require('../data/bucket').EXTENT;
+var Tile = require('../source/tile');
+var pixelsToTileUnits = new Tile().pixelsToTileUnits;
```

Comment:
We should move `pixelsToTileUnits` to a separate module (i.e.`../source/pixels_to_tile_units`). The way this is written, we have created unnecessary coupling between `draw_background` and internal implementation details of `Tile`.